README.md

```diff
@@ -1,65 +1,117 @@
 
-## Table of contents
+## Get involved!
+Check out the [Working Group meeting details](https://github.com/hotosm/tasking-manager/wiki/TM-Working-Group-Meeting-Details), review the [Roadmap](https://github.com/hotosm/tasking-manager/projects/1), and review our [Contributor guidelines](https://github.com/hotosm/tasking-manager/blob/develop/CONTRIBUTING.md).
 
-1 [Intro](#Intro)
+## Architecture
 
-2 [Client Development](#ClientDevelopment)
+The Tasking Manager is composed of two parts:
+* **Client**: A front-end user interface, called the client. It's built using AngularJS.
+* **Server**: A back-end database and API built using python3.
 
-3 [Server Development](#ServerDevelopment)
+The two parts can be developed independently of each other.
 
-4 [Dev Ops](#DevOps)
+## Client
+The client is the front-end user interface of the Tasking Manager. If you're interested in developing the client alone, you can build it using `gulp`, without having to worry about the server set up. You can point the client at a non-local API url (e.g. a staging environment), by changing the environment in `client/taskingmanager.config.json`. All the files pertaining to the client are available in the `client/` directory. 
 
-5 [Localisation](#Localisation)
-
-## <a name="Intro">Intro</a>
-The app is split into a Client (AngularJS) and Server (Python) structure.  Each can be developed independently of each other.  See below for instructions on how to set up your dev environment.
-
-[See our FAQ if you hit any problems getting setup](https://github.com/hotosm/tasking-manager/wiki/Dev-Environment-FAQ)
-
-**Looking to get involved in development?** Check out the [Working Group meeting details](https://github.com/hotosm/tasking-manager/wiki/TM-Working-Group-Meeting-Details), review the [Roadmap](https://github.com/hotosm/tasking-manager/projects/1), and review our [Contributor guidelines](https://github.com/hotosm/tasking-manager/blob/develop/CONTRIBUTING.md).
-
-## <a name="ClientDevelopment">Client Development</a>
-### Global Dependencies
-Following must be available locally:
-
-* NodeJS LTS v6+ [NodeJS LTS install here](https://nodejs.org/en/)
-* [Gulp](http://gulpjs.com/) is used to run and build the app, install globally as follows:
-    * ```npm install gulp -g```
-
-### App Dependencies
-You will now have to install all the app dependencies using [NPM](https://www.npmjs.com/)
+**Dependencies**
+The following dependencies must be available _globally_ on your system:
+* Download and install [NodeJS LTS v6+](https://nodejs.org/en/) and [NPM](https://www.npmjs.com/)
+* Install [Gulp](http://gulpjs.com/).
+  * `npm install gulp -g`
+* Install [Karma](https://karma-runner.github.io/1.0/index.html)
+  * `npm install -g karma karma-jasmine karma-chrome-launcher`
 
+**Build**
+Once you have the above dependencies, install and run the client dependencies using `npm` and `gulp`
 ```
 cd client
 npm install
+gulp run
```

Comment:
I would change this to `gulp build`. Then you could add a section under tests for `**Run**`